### DISCUSSION TITLE
YOURLS in subdirectory with Nginx Webserver

### DISCUSSION THREAD

**Discussion by bnfone** [YOURLS/YOURLS#3462] [Q&A]
Hi there, 
I would like to install YOURLS in a subdirectory on my Webserver to keep things clean. My goal is to get YOURLS working and serving from the root domain / directory - with NGINX. I found a guide on the official documentation [here](https://docs.yourls.org/guide/server-configuration.html#give-yourls-its-own-directory), so it seems possible for Apache.
My thoughts were to do exactly the same for NGINX. I don't know much about Webservers, so I simply used a ```.htaccess``` to ```web.conf```(NGINX) [convert tool](https://winginx.com/en/htaccess) but this didn't worked.

In a nutshell, I installed yourls in ```sho.rt/yourls``` and I want it to serve from ```sho.rt```. 

**Do you have any suggestions how I can do this?**

Thank you in advance :)

**Comment by LeoColomb** [comment]
https://docs.yourls.org/guide/server-configuration.html#nginx
